### PR TITLE
Chapter 10: Updating, showing, and deleting users

### PR DESCRIPTION
**Pull request by huphc1024**
chap10

### CODE REVIEW

**Comment by huphc1024** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -1,22 +1,46 @@
 class User < ApplicationRecord
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
 
+  attr_accessor :remember_token
+
   has_secure_password
 
+  validates :password, presence: true, length: {minimum: 6}, allow_nil: true
```
Comment (dothidiemthao) [inline app/models/user.rb]: 6 settings

Comment (huphc1024) [reply app/models/user.rb]: đã fix ạ

**Comment by huphc1024** [comment]
ready

**Comment by huphc1024** [comment]
ready

File: test/integration/users_edit_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UsersEditTest < ActionDispatch::IntegrationTest
```
Comment (VuThiTranVan) [inline test/integration/users_edit_test.rb]: xóa file này

**Comment by VuThiTranVan** [comment]
merged

**Pull request merged**